src/services/mongoDomainService.ts

```diff
@@ -38,40 +35,40 @@ export class MongoDomainService extends DomainService<MongoDbService> {
   async getSubdomains(
     id: string,
     findOptions: Maybe<DomainFindOptions>
-  ): Promise<PaginationResponse<Domain>> {
+  ): Promise<Domain[]> {
```

Comment:
shouldn't this and all others be ValidDomain[]?

Reply:
Yeah, its just not merged with develop branch yet

Reply:
It should be returning DomainDto now.